components/notification/index.tsx

```diff
@@ -101,7 +101,7 @@ export interface ArgsProps {
   duration?: number;
   icon?: React.ReactNode;
   placement?: NotificationPlacement;
-  style?: string;
+  style?: object;
```

Comment:
React.CSSProperties

https://github.com/ant-design/ant-design/blob/9a5894c46ee9fe48644af72da59cc70c83c285ec/components/button/button.tsx#L51

Reply:
Ah, I see, will use that, thanks!

Reply:
@afc163 one question: how to reference this type in the documentation?

Reply:
[React.CSSProperties](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e434515761b36830c3e58a970abf5186f005adac/types/react/index.d.ts#L794) would be nice.

`[React.CSSProperties](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e434515761b36830c3e58a970abf5186f005adac/types/react/index.d.ts#L794)`